frontend/src/scenes/instance/SystemStatus/index.tsx

```diff
@@ -20,12 +22,13 @@ export function SystemStatus(): JSX.Element {
     const { tab, error, systemStatus } = useValues(systemStatusLogic)
     const { setTab } = useActions(systemStatusLogic)
     const { preflight, siteUrlMisconfigured } = useValues(preflightLogic)
+    const { user } = useValues(userLogic)
 
     return (
         <div className="system-status-scene">
             <PageHeader
-                title="System Status"
-                caption="Here you can find all the critical runtime details about your PostHog installation."
+                title="Instance settings"
```

Comment:
This was out of sync with breadcrumbs, and somewhat incorrect as only one tab is settings. Renamed to "Instance status" for consistency. (But we should rename the sidebar item back to "Instance settings" if we make it point to _only_ a settings page)

<img width="283" alt="Screen Shot 2022-02-17 at 12 40 48" src="https://user-images.githubusercontent.com/4550621/154474242-94388b7a-2f7b-4bdb-b627-7c379a635404.png">